### DISCUSSION TITLE
is there a way that to install jobe server not in html folder?  eg: html/projecta/jobe

### DISCUSSION THREAD

**Discussion by azizikri** [trampgeek/jobe#70] [Q&A] [answered]
in the tutorial jobe can only installed in html folder, can jobe be installed at a custom location?

**Comment by trampgeek** [comment]
The installer will give you a warning if you install anywhere other than /var/www/jobe or /var/www/html/jobe. I suggest you try installing elsewhere and report back on what happens. Assuming you're using Jobe from CodeRunner you will of course also have to ensure that the address you give for the Jobe server in the CodeRunner settings takes you to the right place, or configure the webserver on Jobe to redirect to your non-standard location. I think that's why I put the warning in the install script, though I don't remember exactly.

**Reply by azizikri** [reply]
hey, sorry for the late response, there are already a webserver in my custom location (var/www/html/project) using nginx, can i just clone the jobe server in the directory?

**Comment by trampgeek** [comment, marked as answer]
Jobe was designed to use Apache. It uses somewhat quirky URLs like http://server.somewhere/jobe/index.php/restapi/languages. nginx processes these differently from Apache, so you can't simply switch between Apache and nginx. You can build a configuration file for nginx to handle those URLs (there's a file nginx.conf included with Jobe that you can use as a starting point) but a much easier approach is to fire up jobeinabox in docker or podman - see [here](https://github.com/trampgeek/jobeinabox). That will give you a jobe server running on a different port (4000 in the readme.md instructions, though you can choose any port you like). Or, if you wish to use port 80 for jobe as well, you can configure nginx to pass all jobe requests on port 80 through to the docker/podman container on port 4000. See [NGINX Reverse Proxy](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/).

**Reply by azizikri** [reply]
hey thanks for the fast response, yeah i just discover jobeinabox, and it's work, thanks a lot tramp
